content/docs/dispensary/prescriptions.md

```diff
@@ -540,3 +540,33 @@ This is done in the Side panel actions section.
 Once a prescription is cancelled, it will still show in the list of prescriptions, but with a status of `CANCELLED`.
 
 ![Example Canceled Prescription](/docs/dispensary/images/prescription-cancelled.png)
+
+## Payment
+
+1. Once you're finished entering the items for the Prescription, click the `Confirm Verified` button.
+
+   ![Insurance Confirm Verified](/docs/dispensary/images/insurance_confirm_verified.png)
+
+2. If there is a non-zero sell price for the items in the prescription, A new window will appear for you to enter the payment details for the patient's prescription.
```

Comment:
Is there a way to configure my system to not show this payment modal?

Reply:
I think this is where global pref comes into play would that be correct for me to say @jmbrunskill

Reply:
thanks - that's what I thought. I thought if a user was reading the docs that's something they would be interested to know.